grade-scripts/trackers-from-requests.js

```diff
@@ -14,6 +14,7 @@ program
     .option('-i, --input <name>', 'The name to use when looking for sites, e.g. "test" will look in "test-sites" (required)')
     .option('-o, --output <name>', 'Output name, e.g. "test" will output files at "test-grades" (required)')
     .option('-f, --file <name>', 'Allow processing a subset of dumped site data, defined in a file')
+    .option('-p, --regenerate-prevalence', 'Regenerate prevalence data from this run')
```

Comment:
I'm not sure whether it's better to have this script regenerate the prevalence data by default, or whether you need to pass a flag to regenerate it.

I've made it so you need to pass a flag to regenerate it.